packages/loaders/src/elements/Spinner.tsx

```diff
@@ -47,19 +47,19 @@ const computeFrames = (
 
 export interface ISpinnerProps extends React.HTMLAttributes<SVGSVGElement> {
   /**
-   * Sets the height and width in pixels. Inherits the parent `font-size` by default
+   * Sets the height and width in pixels. Inherits the parent's font size by default.
    **/
   size?: string;
   /**
-   * Sets the speed of the loading animation in milliseconds
+   * Sets the length of the animation cycle in milliseconds
    **/
   duration?: number;
   /**
-   * Sets the fill color. Inherits the parent font `color` by default
+   * Sets the fill color. Inherits the parent's `color` by default.
    **/
   color?: string;
   /**
-   * Delays showing the loader to prevent a quick render flash during normal loading times. Time is in milliseconds
+   * Delays displaying the loader to prevent a quick render flash during normal loading times
```

Comment:
Should ~quick~ be removed per your previous description?

Reply:
Agreed